src/entities/person/possible-duplicates.ts

```diff
@@ -0,0 +1,82 @@
+import { Universe } from 'src/universe'
+import { EntityRawPayload, UniverseEntityOptions } from '../_base'
+import { PersonRawPayload, PersonPhonenumberRawPayload } from './person'
+import { ChannelUserRawPayload } from './channel-user'
+import { BaseError } from '../../errors'
+
+export interface PossibleDuplicatesRawPayload extends EntityRawPayload {
+  readonly person_id: string
+  readonly strategies: PossibleDuplicatesStrategies
+}
+
+interface PossibleDuplicatesStrategies {
+  global_phonenumber?: StrategiesResult[]
+  name_exact_matches?: StrategiesResult[]
+}
+
+interface StrategiesResult {
+  document: StrategiesDocument
+  kind: string
+}
+
+interface StrategiesDocument {
+  person: Pick<PersonRawPayload, 'id'| 'kind' | 'name'>
+  channel_user?: Pick<ChannelUserRawPayload, 'id' | 'name' | 'phone'>
+  phone_number?: Pick<PersonPhonenumberRawPayload, 'id' | 'value'>
+}
+
+interface PossibleDuplicationOptions extends UniverseEntityOptions {
+  rawPayload: PossibleDuplicatesRawPayload
+}
+
+export class PossibleDuplication {
```

Comment:
making this a class, instead of interface is kinda optional. Class would only be needed to eventually make subsequent actions on that duplicate. For now that is likely not planned, but a case would be making the merge of the resulting person from this instance. Since the UX flow is rather to open a modal and configure that merge, that case though is not strong

Reply:
👍 ok let's make it an interface then as I don't have strong argument either for it to be a class